app/views/themes/institutional_repository/layouts/hyrax.html.erb

```diff
@@ -10,25 +10,25 @@
     <div class="skip-to-content">
       <%= link_to "Skip to Content", "#skip-to-content" %>
     </div>
-    <%= render '/masthead' %>
+    <%= render '/masthead', placement_class: nil %>
     <%= content_for(:navbar) %>
     <%= content_for(:precontainer_content) %>
     <div id="content-wrapper" class="container" role="main">
       <%= render '/flash_msg' %>
-      <%= render_breadcrumbs builder: Hyrax::BootstrapBreadcrumbsBuilder %>
+      <%= render_breadcrumbs builder: Hyrax.config.breadcrumb_builder %>
       <% if content_for?(:page_header) %>
         <div class="row">
-          <div class="col-xs-12 main-header">
+          <div class="col-12 main-header">
             <%= yield(:page_header) %>
           </div>
         </div>
       <% end %>
 
-        <a name="skip-to-content" id="skip-to-content"></a>
-        <%= content_for?(:content) ? yield(:content) : yield %>
-
+      <a name="skip-to-content" id="skip-to-content"></a>
+      <%= render 'shared/read_only' if Flipflop.read_only? %>
```

Comment:
Reading Note: Interesting add

Reply:
Trying to match Hyrax again
https://github.com/samvera/hyrax/blob/double_combo/app/views/layouts/hyrax.html.erb#L27-L28